### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by RToister**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,66 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list | None = None
```
Comment (vasylhnatiuk) [inline app/main.py:10]:
```suggestion
                 coords: list[int] | None = None
```

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,66 @@ class Cargo:
...
+        self.coords[2] += step
+
+    def go_down(self, step: int = 1) -> None:
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(
+            self, name: str,
+            weight: int, coords: list | None = None,
```
Comment (vasylhnatiuk) [inline app/main.py:54]: same here

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,66 @@ class Cargo:
...
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] | None = None
+                 ) -> None:
+        if coords is None:
```
Comment (vasylhnatiuk) [inline app/main.py:38]: use or here as well

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (RToister) [reply app/main.py:54]: Thanks, will fixed

**Comment by RToister** [comment]
Hello. I try to understand why I can't make one more pull request after first (  
for Solution3

**Pull request closed without merging**